cumulusci/cumulusci.yml

```diff
@@ -1000,6 +1002,10 @@ project:
                 5:
                     class_path: cumulusci.tasks.release_notes.parser.GithubLinesParser
                     title: Deleted Metadata
+                6:
+                    class_path: cumulusci.tasks.release_notes.parser.InstallLinkParser
+                    title: Installation Links
+                    skip_parent_pr: True
```

Comment:
Why is this needed? There won't be a version_id when parent release notes are generated, so the InstallLinkParser won't produce any content.

Reply:
I don't remember clearly, but vaguely remember it was something to do with how the two tasks are initializing the generator and parsers.  There's likely a better way to implement this.